### PR TITLE
create calcule and operate methods

### PR DESCRIPTION
**Pull request by rida-elbahtouri**
### In this branch I:
- Added calculate logic
- operate method to return new value based on the operation selected

### CODE REVIEW

File: src/Logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      return num1.plus(num2).toString();
+    case '*':
+      return num1.times(num2).toString();
+    case '÷':
+      if (num2.toString() === '0') {
+        return "can't divide by zero";
+      }
+      return num1.div(num2).toString();
+    case '%':
+      return num1.mod(num2).toString();
```
Comment (bellom) [inline src/Logic/operate.js]: When it comes to `%` we only need one operand.
example: click `10` then `%` = 0.01

example: `num1 = (num1 *= 0.01).toString();`

File: src/Logic/calculate.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  } else if (!operations.includes(buttonName)) {
+    if (total === null) {
+      total += buttonName;
+    } else {
+      next += buttonName;
+    }
+  } else if (operations.includes(buttonName)) {
+    operation = buttonName;
+  }
+
```
Comment (bellom) [inline src/Logic/calculate.js]: let's try to add a case where a user clicks `.` we should return `0.` on the screen.

example: If click `.` I should see `0.`

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @rida-elbahtouri  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Comment by rida-elbahtouri** [comment]
most calculators when you click on % you need to add the other number to the result like 4000 % 6 should get 240. because 240 present 6% of the 4000.

**Review by Torres-ssf — APPROVED** [review]
### Milestone approved :+1: 
Hi @rida-elbahtouri :hand: 
Great job implementing the requested changes.
You may merge the PR now.

Happy Coding :computer: 
**Sergio Torres**

**Pull request merged**